app/apps/server/orchestrator.js

```diff
@@ -190,17 +190,6 @@ settings.addGroup('General', function() {
 			public: true,
 			hidden: false,
 		});
-
```

Comment:
I feel like removing the ability to disable the game center is the wrong way to do this. Instead, this PR could be a simple one line change at like 194.

```
this.add('Apps_Game_Center_enabled', false, {
```

would become

```
this.add('Apps_Game_Center_enabled', true, {
```

Reply:
Hi, @wreiske ! You're right because we're using `PrivateSettingsCachedCollection` for settings cache internally. For new deployed instance this PR works. For existing instance, we need to think how to write `migrations` to adapt the database changes instead of leaving the unused/dead code in the codebase.

Reply:
Updated, thanks for your kindly reminder ;)

Reply:
Hey @wreiske, thanks for the input! After some discussion, we realized that removing the setting is actually harmless, considering that the workspace still needs to have "game apps" installed and enabled for the game center to be presented - meaning that if the admin doesn't want games to be played, they can just disable/uninstall the apps.